### PR TITLE
#166151258 Add support for Drop Sets

### PR DESCRIPTION
**Pull request by wekesa931**
### What does this PR do?
Adds a feature to support Drop Sets

### Description of Task to be completed?
When creating a workout, when a user wants to create an exercise in the workout schedule, they should be able to add drop sets for the exercise.
### How should this be manually tested?
Clone the repository to your local branch
```
$ git clone https://github.com/andela/wg-baqi.git
```
$ cd to the `wg-baqi` directory, create a virtual environment and install the dependencies
```
$ cd wg-baqi
$ virtualenv venv
$ source venv/bin/activate
$ pip install -r requirements.txt
```
Activate bootstrap, make migrations and run the server
```
$ bower install
$ invoke bootstrap-wger --settings-path ./wger/settings.py --no-start-server
$ python manage.py runserver
```
- Log in to the application
- Navigate to the route `http://127.0.0.1:8000/en/workout/`
- Click on `Add exercises to this workout day`
Screenshot 2019-06-13 at 11 32 43

- Add an exercise, select the number of sets.
Screenshot 2019-06-13 at 11 30 49

- Check the `Drop set` checkbox
- Determine the percentage drop, cycles, and weight.
- Calculate the drop set
Screenshot 2019-06-13 at 11 31 04

### What are the relevant pivotal tracker stories?
[#166151258](https://www.pivotaltracker.com/story/show/166151258) 

Before working on the feature

Screenshot 2019-06-12 at 20 01 56

After working on the feature
Screenshot 2019-06-12 at 19 56 20

### CODE REVIEW

File: wger/manager/templates/set/add.html [hunk 1]
```diff
@@ -4,17 +4,20 @@
...
+{% block title %}
+{% with day_name=day.description %}
+{% blocktrans %}Add exercises to day "{{day_name}}"{% endblocktrans %}{% endwith %}{% endblock %}
 
 
 <!--
         Header
 -->
-{% block header %}
+{% block header %}\
```
Comment (wangonya) [inline wger/manager/templates/set/add.html]: Is the `\` here part of the code? 😅🤔

File: wger/manager/templates/set/add.html [hunk 2]
```diff
@@ -62,6 +65,48 @@
...
+            <strong>Percentage</strong> - Drop the weight by a percentage.<br/>
+            <strong>Cycles</strong> - Number of cycles on the same weight.<br/>
+            <strong>Weight</strong> -  Size of weight to start with.<br/>
+        {% endblocktrans %}
+        </span>
+        </div>
+        <div style="max-width: 45%; margin: auto;">
+          <div class="form-row">
+            <div class="form-group">
+                <label for="exampleFormControlSelect1">Percentage Drop</label>
```
Comment (wangonya) [inline wger/manager/templates/set/add.html]: The `for=""` in the `<label>` here should match the `id=""` of the `<select>` it is *labelling*.

**Example:**
You have:
```
...
<label for="exampleFormControlSelect1">Percentage Drop</label>
    <select class="form-control" id="percentage-change" ...>
...
```

It should be:
```
...
<label for="percentage-change">Percentage Drop</label>
    <select class="form-control" id="percentage-change" ...>
...
```

Do the same for the rest of the elements in the form.

Comment (wangonya) [reply wger/manager/templates/set/add.html]: For more reference on the same, see [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#Attributes) and [this](https://stackoverflow.com/questions/18432376/what-does-for-attribute-do-in-html-label-tag?answertab=votes#tab-top)

File: wger/manager/templates/set/formset.html [hunk 3]
```diff
@@ -1,5 +1,9 @@
 {% load wger_extras %}
 {% load i18n %}
+{% load staticfiles %}
+
```
Comment (ceexon) [inline wger/manager/templates/set/formset.html]: Why do you have blank spaces here
Kindly, give us a reason for this.

**Review by wangonya — CHANGES_REQUESTED** [review]
Nice work on the feature. Have a look at the minor issues noted.

**Review by jmusila — COMMENTED** [review]
Tested locally, it is working 👍

File: wger/manager/templates/set/formset.html (hunk 3 above)
Comment (wekesa931) [reply wger/manager/templates/set/formset.html]: I have made the adjustments. I believe it is okay now

**Review by wangonya — APPROVED** [review]

**Pull request merged**